examples/ibm-cloudant-database/modules/instance/README.md

```diff
@@ -0,0 +1,111 @@
+# IBM Cloud Cloudant - Terraform Module
+
+This is a collection of modules that make it easier to provision cloudant instance and assign service prolicy, create a database:
+* [instance](modules/instance)
+* [service-policy](modules/service-policy)
+* [database](modules/database)
+
+## Compatibility
+
+This module is meant for use with Terraform 0.13 (and higher).
+
+## Usage
+
+Full examples are in the [examples](./examples/) folder, but basic usage is as follows for creation of logdna instance & key:
```

Comment:
```suggestion
Full examples are in the [examples](./examples/) folder, but basic usage is as follows for creation of a Cloudant instance & key:
```

Reply:
Good catch! Now we know where this (rather none standard) README was copied from 😄